src/email/services/mailing.service.ts

```diff
@@ -19,4 +27,30 @@ export class MailingService {
       throw new MailSendingException();
     }
   }
+
+  public async sendMailsToSubscribers(payload: IRate): Promise<void> {
+    const { currencyCode, date, rate } = payload;
+
+    const mailMessage = this.mailHtmlTemplate(currencyCode, rate, date);
+    const mailSubject = 'Exchange rate USD to UAH';
+
+    const subscribers = await this.emailService.findAll(true);
+    const mailPromises = subscribers.map((subscriber) =>
+      this.sendMail({
+        html: mailMessage,
+        subject: mailSubject,
+        to: subscriber.email,
+      }),
+    );
+
+    await Promise.allSettled(mailPromises);
+  }
+
+  private mailHtmlTemplate(
+    currencyCode: string,
+    rate: number,
+    date: string,
+  ): string {
+    return `<p>Current rate ${currencyCode} to UAH - ${rate}. Date: ${date}</p>`;
+  }
```

Comment:
At the moment, this service is less flexible than the others:
- Starting with clearly stated currencies
- Up to sending emails to active subscribers only